core/src/test/java/org/apache/iceberg/PartitionStatsHandlerTestBase.java

```diff
@@ -696,6 +769,12 @@ private DeleteFile commitPositionDeletes(Table testTable) {
     return posDelete;
   }
 
+  private DeleteFile commitDVs(Table testTable, DataFile dataFile) {
```

Comment:
the name is misleading, since it only adds a single DV and not multiple ones. I don't think it's even worth adding this method, since it is only used in one single place, so maybe just inline this?

Reply:
updated.